packages/common/src/SolcoverConfig.ts

```diff
@@ -1,10 +0,0 @@
-export const SolcoverConfig = {
```

Comment:
this is not truffle specific. it actually looks like the last time our coverage ran was feb of this year: https://coveralls.io/github/UMAprotocol/protocol?branch=master

<img width="1158" alt="image" src="https://user-images.githubusercontent.com/12886084/135275925-9ee21ab3-1d00-457d-8bd3-b7f14fcce17a.png">


we should consider adding a task to get this back up and running.